styles/youtube/catppuccin.user.css

```diff
@@ -425,1067 +442,324 @@
+        background-color: @surface2 !important;
+      }
     }
 
-    .iv-card-content > :first-child,
-    .iv-card h2,
-    .ytp-cards-teaser .ytp-cards-teaser-text {
-      color: @text !important;
-    }
+    /* Panels, popups, tooltips */
 
     .ytp-tooltip-text {
       background: @surface0 !important;
       color: @text;
       text-shadow: none !important;
     }
 
-    .iv-card:hover .iv-card-primary-link,
-    .ytp-ce-website-title {
-      color: @accent-color !important;
-    }
-
     .ytp-popup {
       background: @surface0 !important;
       text-shadow: none;
     }
 
-    tp-yt-paper-dialog {
-      background-color: @base !important;
+    .ytp-panel-menu,
+    .ytp-panel-header,
+    .ytp-panel-footer,
+    .ytp-menuitem-label,
+    .ytp-premium-label,
+    .ytp-menuitem-content,
+    .ytp-menuitem-label-count,
+    .ytp-menu-label-secondary {
+      color: @text;
     }
 
-    /* live badges */
-    .badge-style-type-live-now-alternate,
-    ytd-thumbnail-overlay-time-status-renderer[overlay-style="LIVE"] {
-      color: @crust !important;
-      yt-icon svg {
-        g path,
-        path {
-          fill: @crust !important;
-        }
-      }
-    }
-    .ytp-live-badge {
-      &::before {
-        background: @subtext1 !important;
-      }
-      &[disabled]::before {
-        background: @accent-color !important;
-      }
+    .ytp-panel-header {
+      border-bottom-color: @surface2;
     }
 
-    /* notification idicator */
-    .yt-spec-icon-badge-shape--type-notification
-      .yt-spec-icon-badge-shape__badge {
-      background-color: @accent-color !important;
-      color: @crust !important;
-    }
-    .yt-spec-icon-badge-shape--style-overlay .yt-spec-icon-badge-shape__icon {
-      color: var(--yt-spec-icon-active-other) !important;
+    .ytp-panel-footer-content-link {
+      color: @accent-color;
     }
-    .yt-spec-icon-badge-shape {
-      color: var(--yt-spec-icon-active-other) !important;
+
+    .ytp-panel-back-button {
+      @svg: escape(
+        '<svg xmlns="http://www.w3.org/2000/svg" width="100%" height="100%" viewBox="0 0 32 32" version="1.1"><path d="m19.41 20.09-4.58-4.59 4.58-4.59L18 9.5l-6 6 6 6z" fill="@{text}"/></svg>'
+      );
+      background-image: url("data:image/svg+xml,@{svg}");
     }
 
-    /* bell next to subscription button */
-    .yt-animated-icon svg {
-      g path,
-      path {
-        fill: @text !important;
+    .ytp-menuitem-toggle-checkbox {
+      background-color: @surface2;
+
+      &::after {
+        background-color: @subtext0;
       }
     }
 
-    /* creator badges */
-    ytd-author-comment-badge-renderer {
-      --yt-basic-background-color: @surface0 !important;
-      --yt-basic-foreground-title-color: @surface0 !important;
-      --ytd-author-comment-badge-background-color: @surface0 !important;
-      --ytd-author-comment-badge-name-color: @text !important;
-      --ytd-author-comment-badge-icon-color: @text !important;
-    }
```

Comment:
removing this breaks 
![image](https://github.com/catppuccin/userstyles/assets/71222764/13766391-b564-47a6-b348-cfa867eaa00e)